tools/pathwaymatcher/pathwaymatcher.xml

```diff
@@ -0,0 +1,332 @@
+                #if $proteoforms.proteoform_match_criteria:
+                  -t proteoform -m "${proteoforms.proteoform_match_criteria}" -i "${proteoforms.input_proteoforms}"
+                #else:
+                  -t proteoform -i "${proteoforms.input_proteoforms}"
+                #end if
+            #end if
+
+
+            ## OUTPUT OPTIONS
+
+            #if $output_options.proteoform_match_criteria:
+                -tlp
+            #end if
+
+            #set $output_graphs_list = str($output_options.output_graphs).split(',')
+
+            #if 'gg' in $output_graphs_list:
+                -gg
+            #end if
+
+            #if 'gu' in $output_graphs_list:
+                -gu
+            #end if
+
+            #if 'gp' in $output_graphs_list:
+                -gp
+            #end if
+
+        2>> $temp_stderr);
+
+        ## We create a zip file to contain graphs files.
+        #if $output_options.output_graphs:
+            touch "emptyfile.txt";
+            zip "graphs.zip" "emptyfile.txt";
+            zip -d "graphs.zip" "emptyfile.txt";
+        #end if
+
+        #if 'gg' in $output_graphs_list:
+            zip -u "graphs.zip" "geneExternalEdges.tsv" "geneInternalEdges.tsv" "geneVertices.tsv";
+        #end if
+
+        #if 'gu' in $output_graphs_list:
+            zip -u "graphs.zip" "proteinExternalEdges.tsv" "proteinInternalEdges.tsv" "proteinVertices.tsv";
+        #end if
+
+        #if 'gp' in $output_graphs_list:
+            zip -u "graphs.zip" "proteoformExternalEdges.tsv" "proteoformInternalEdges.tsv" "proteoformVertices.tsv";
+        #end if
+
+
+        exit_code_for_galaxy=\$?;
+        cat $temp_stderr 2>&1;
+        (exit \$exit_code_for_galaxy)
+]]>
+    </command>
+    <inputs>
+
+        <section name="genetic_variants" expanded="true" title="Genetic variants">
+
+          <param format="txt" name="input_rsid" type="data" label="SNP rsId list" optional="true"
+            help="The file contains one rsid identifier as defined in dbSNP[1] on each row.
+            The list must be ordered by chromosome and base pair (bp). The list must not have duplicates.
+            All rsids must appear in the human assembly GRCh37.p13. "/>
+
+          <param format="txt" name="input_chrbp" type="data" label="Chromosomes and base pairs" optional="true"
+            help="Genetic variants can also be represented using the chromosome and the base pair numbers.
+            The input should be sorted by chromosome number and then by base pair.  "/>
+
+          <param format="txt" name="input_vcf" type="data" label="Variant Call Format Specification" optional="true"
+            help="The input follows the Variant Call Format Specification[2] v4.3.
+            It also allows the possibility to specify only the first 4 columns in the data section of the file:
+            CHROM, POS, ID, REF.  "/>
+
+        </section>
+
+
+        <section name="genes" expanded="true" title="Genes">
```

Comment:
Does it make sense to have a section with just one option?

Reply:
I think it separates visual and clearly different functional areas of PathwayMatcher app... and it looks far better :-)

Reply:
It makes the wrapper more complicated imho. If the style is sub-optimal we should fix this in Galaxy for all tools and keep the look and feel the same for all wrappers. No?

Reply:
Well, I think the wrapper is still quite simple and (internally) very repetitive, so the additional complexity is almost zero and the visual hierarchy and functional separation (in the same way than the app itself) is an improvement that compensates it... 
But, of course, this may be quite subjective... about the functional separation, I think it's actually an improvement, but any designer may have different opinions about this visual style and aesthetics...

Reply:
To make this a little bit less subjective we tried to develop standards for UX and tools styles here: http://galaxy-iuc-standards.readthedocs.io/en/latest/